src/Tribe/Importer/File_Importer_Venues.php

```diff
@@ -9,23 +9,26 @@ class Tribe__Events__Importer__File_Importer_Venues extends Tribe__Events__Impor
 
 	protected function match_existing_post( array $record ) {
 		$name = $this->get_value_by_key( $record, 'venue_name' );
-		$id   = $this->find_matching_post_id( $name, Tribe__Events__Main::VENUE_POST_TYPE );
+		$id   = $th is->find_matching_post_id( $name, Tribe__Events__Main::VENUE_POST_TYPE );
```

Comment:
this looks like a bug

Reply:
it is ! fixed now tho, ready for another review 🔍